tests/active_learning/test_ScorerSemanticSegmentation.py

```diff
@@ -0,0 +1,90 @@
+import unittest
+
+import numpy as np
+
+import lightly
+from lightly.active_learning.scorers import ScorerSemanticSegmentation
+from lightly.active_learning.scorers import ScorerClassification
+
+class TestScorerSemanticSegmentation(unittest.TestCase):
+
+    def setUp(self):
+
+        self.N = 100
+        self.W, self.H, self.C = 32, 32, 10
+
+        # the following data should always pass
+        self.dummy_data = np.random.randn(self.N * self.W * self.H, self.C)
+        self.dummy_data /= np.linalg.norm(self.dummy_data, axis=-1)[:, None]
```

Comment:
Why are you dividing by the norm instead of the sum? This way the probabilities do not add up to 1.

Reply:
Because I'm a stupid man 😄